internal/pkg/agent/cmd/status.go

```diff
@@ -19,16 +19,19 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/jedib0t/go-pretty/v6/list"
+
 	"github.com/elastic/elastic-agent/internal/pkg/agent/errors"
 	"github.com/elastic/elastic-agent/internal/pkg/cli"
 )
 
 type outputter func(io.Writer, interface{}) error
 
 var statusOutputs = map[string]outputter{
-	"human": humanStateOutput,
-	"json":  jsonOutput,
-	"yaml":  yamlOutput,
+	"human":      humanOutput,
+	"human_full": humanFullOutput,
```

Comment:
What about just `full`? Is the `human_` prefix needed? Just would like to keep it the options super simple.

Reply:
Changed to just `full`